client/src/main/java/academy/konrad/group/battleships/userinterface/Main.java

```diff
@@ -16,17 +20,23 @@ public static void main(String[] args) {
   }
 
   @Override
-  public void start(Stage primaryStage) throws Exception {
+  public void start(Stage primaryStage) throws IOException {
+    loggerSetup();
     renderView(primaryStage);
     Connection.initialize();
   }
 
+  private void loggerSetup() {
+    //sciezka na serwerze
+    Configurator.defaultConfig().writer(new FileWriter("/home/admin/logi/info_klient.txt"), Level.INFO)
```

Comment:
LOGGER ERROR: Failed to activate configuration (java.io.FileNotFoundException: /home/admin/logi/info_klient.txt (Nie ma takiego pliku ani katalogu))

Proponuję zmienić ścieżkę na taką, która jest dostępna niezależnie od maszyny, np. ~/logi/info_klient.txt i ~/logi/bledy_klient.txt

Reply:
Ok racja, poprawione